service/assessment/assessment.go

```diff
@@ -318,15 +325,19 @@ func (svc *Service) handleEvidence(ev *evidence.Evidence, resourceId string) (er
 		return newError
 	}
 
-	// Get Evidence Store stream
-	channelEvidenceStore, err := svc.evidenceStoreStreams.GetStream(svc.evidenceStoreAddress.target, "Evidence Store", svc.initEvidenceStoreStream, svc.evidenceStoreAddress.opts...)
-	if err != nil {
-		err = fmt.Errorf("could not get stream to evidence store (%s): %w", svc.evidenceStoreAddress.target, err)
-		log.Error(err)
+	// Send evidence via Evidence Store stream if hasEvidenceStore is true
+	if svc.hasEvidenceStore {
```

Comment:
Instead of new variable, we could also just check if `svc.evidenceStoreStreams` is nil which is directly set through the `Without` option. Would be less code but also maybe less readable

Reply:
I would keep the extra option, less error prone